catalog-info.yaml

```diff
@@ -0,0 +1,82 @@
+apiVersion: backstage.io/v1alpha1
+kind: System
+metadata:
+  name: rhdh
+  title: RHDH
+  description: Red Hat Developer Hub is an enterprise-grade Internal Developer Portal based on Backstage.
+  annotations:
+    backstage.io/techdocs-ref: dir:.
+    github.com/project-slug: redhat-developer/rhdh
+    github.com/team-slug: redhat-developer/rhdh-team
+    quay.io/repository-slug: rhdh-community/rhdh
+    sonarqube.org/project-key: janus-idp_backstage-showcase
```

Comment:
This is after the migration still the current sonarqube key. See statusbar link on this screenshot:

![image](https://github.com/user-attachments/assets/2be2e4a6-fda3-4e86-872b-3e08558be10c)